JuiceMaker/JuiceMaker/Controller/JuiceMakerViewController.swift

```diff
@@ -6,7 +6,7 @@
 
 import UIKit
 
-final class JuiceMakerViewController: UIViewController {
+final class JuiceMakerViewController: UIViewController, FruitStoreDelegate {
```

Comment:
주로 가독성을 위해 extension으로 확장하여 프로토콜을 채택해주기도 합니다.
```suggestion
final class JuiceMakerViewController: UIViewController { }

extension JuiceMakerViewController: FruitStoreDelegate { }
```

Reply:
`FruitStoreDelegate` 채택을 `extension`으로 옮기고, 명세 메서드도 해당 클로저 안으로 이동했습니다.